lib/kovid/cli.rb

```diff
@@ -56,7 +56,16 @@ def compare(*name)
 
     desc 'states STATE STATE', 'Returns full comparison table for the given states. Accepts multiple states.'
     def states(*states)
-      puts Kovid.states(states)
+      # This ensures this command is case insensitive.
+      downcased_states = states.map(&:downcase)
```

Comment:
This was the key to fixing state comparison and making the comparison case insensitive. Without this `list.include?(state['state'].downcase)` [here](https://github.com/siaw23/kovid/pull/95/files#diff-0ab52c92c4045a1db9f29de46b5d70d5R197) was never true if a user did `kovid states Florida Tennessee`, as the `downcase` call on the fetched state data would never match the capitalized state that was passed as an argument in the terminal.